### ISSUE TITLE
Add a configuration option to control whether `RUN_DISPLAY_URL` uses pipeline-graph-view

### ISSUE THREAD

**Issue by nre-ableton** [jenkinsci/pipeline-graph-view-plugin#448]
### What feature do you want to see added?

My company would like to migrate to the pipeline-graph-view plugin, and we've been using it experimentally for some months now. While the plugin is great, we understand that it's still under development and there are some areas that don't work so well with some of our pipelines. Also, our users are used to Blue Ocean and we need to do a bit of preparation to get them used to the new UI.

It would be nice if we could run both pipeline-graph-view and Blue Ocean in parallel and just point our more experienced/adventurous users to the pipeline graph view, but unfortunately when both of these plugins are installed, URLs like `RUN_DISPLAY_URL` point to the pipeline graph view and not Blue Ocean. I don't see a way to configure this, but I suspect it is possible since I ran across this line when looking through the code:

https://github.com/jenkinsci/pipeline-graph-view-plugin/blob/f83608aa073a71aa405280eb7330714487161054/src/main/java/io/jenkins/plugins/pipelinegraphview/PipelineGraphDisplayURLProvider.java#L10

However, it's not at all obvious to me how to configure the provider. It would be nice if this were documented, or if there were an option to read the precedence from the JCasC configuration rather than being hardcoded at 50.

### Upstream changes

N/A

### Are you interested in contributing this feature?

If someone can point me in the right direction, then yes!

**Comment by timja** [comment]
Duplicate of https://github.com/jenkinsci/pipeline-graph-view-plugin/issues/378

You can configure it on the Appearance system page:

<img width="1253" alt="image" src="https://github.com/jenkinsci/pipeline-graph-view-plugin/assets/21194782/dbc25c36-3a35-45ce-8a1b-fbf96b377a5d">

Or on a per user basis:

<img width="1131" alt="image" src="https://github.com/jenkinsci/pipeline-graph-view-plugin/assets/21194782/d477a29d-4f7b-4ab1-b0d3-84c2853e062b">

**Comment by nre-ableton** [comment]
Thanks @timja, and sorry for the noise!

**Comment by nre-ableton** [comment]
Hmm, annoying that Blue Ocean doesn't show up in that list. Does it not implement `consoleUrlProvider`, I wonder? I'll do some digging...

**Comment by timja** [comment]
Ah you're asking about the notification url that's on the same page
